jeoparpy/ui/maingame/podiapanel.py

```diff
@@ -72,11 +73,17 @@ def update(self, gameState, gameData):
 
     def _init_background(self):
         img = pygame.image.load(IMAGES['rPanelBG']).convert()
+
         sizeScalar = float(self.size[1]) / img.get_size()[1]
+
+        # Change the scalar depending on the number of players
+        # NOTE: I don't really know why this involves a 3
```

Comment:
I don't know why this has a three in the equation, but that is what works. When the number of players is 3, `sizeScalar` is unchanged